src/metabase/events/view_log.clj

```diff
@@ -1,6 +1,7 @@
 (ns metabase.events.view-log
   "This namespace is responsible for subscribing to events which should update the view log and view counts."
   (:require
+   [grouper.core :as grouper]
```

Comment:
I just recalled the suggestion in the Slack thread to hide grouper behind our own thin namespace wrapper. I think we should still do this, especially to ensure that the promises never leak out into our own code.